src/libs/antares/inifile/inifile.cpp

```diff
@@ -237,29 +238,20 @@ uint IniFile::Section::size() const
 
 bool IniFile::open(const AnyString& filename, bool warnings)
 {
-    // clear
     clear();
-    pFilename = filename;
+    std::string filePath = filename.to<std::string>();
+    const unsigned int lineMaxSize = 2048;
```

Comment:
```suggestion
```

Reply:
Declaration of **lineMaxSize** was removed